### PR TITLE
Add verbose option

### PR DESCRIPTION
**Pull request by tamirkiviti13**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -70,14 +70,32 @@ const downloadTraceeCommercial = async () => {
...
+    traceeCommand.push('-v')
+  }
+
+  if (flags.quiet) {
+    traceeCommand.push('-q')
+  }
+
+  if (flags.logFile) {
+    traceeCommand.push('--log-file')
+    traceeCommand.push(`"${flags.logFile}"`)
```
Comment (orizerah) [inline src/index.ts:86]: 2 checks:
1. That the file doesn't already exist
2. That the folder does exist

File: src/index.ts [hunk 2]
```diff
@@ -114,14 +132,20 @@ async function run(): Promise<void> {
...
-    }
+    // let repoPath = core.getInput('repo-path')
+    // if (repoPath === '') {
+    //   repoPath = '.'
+    // }
+
+    // const verbose = core.getInput('verbose') === 'true'
+    // const quiet = core.getInput('quiet') === 'true'
+    // const logFile = core.getInput('log-file')
+    // const accessToken = core.getInput('access-token')
```
Comment (orizerah) [inline src/index.ts:143]: delete

File: src/index.ts [hunk 3]
```diff
@@ -70,14 +70,38 @@ const downloadTraceeCommercial = async () => {
...
+  }
+
+  if (flags.quiet) {
+    traceeCommand.push('-q')
+  }
+
+  if (flags.logFile) {
+    if (validateLogFilePath(flags.logFile)) {
+      traceeCommand.push('--log-file')
+      traceeCommand.push(`"${flags.logFile}"`)
```
Comment (davidsalame1) [inline src/index.ts:87]:
```suggestion
      traceeCommand.push('--log-file', `"${flags.logFile}"`)
```

File: src/inputs.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    quiet: core.getInput('quiet') === 'true',
+    logFile: core.getInput('log-file'),
+    repoPath: repoPath || '.',
+    accessToken: core.getInput('access-token'),
+    aquaKey: core.getInput('aqua-key'),
+    aquaSecret: core.getInput('aqua-secret')
+  }
+}
+
+export const validateLogFilePath = (logFilePath: string): boolean => {
```
Comment (davidsalame1) [inline src/inputs.ts:21]: Either throw an exception or change the name to `isLogFilePathValid`

**Pull request merged**